src/vs/base/browser/ui/positronComponents/splitters/verticalSplitter.tsx

```diff
@@ -53,27 +54,14 @@ type VerticalSplitterCollapseProps = | {
 type VerticalSplitterProps = VerticalSplitterBaseProps & VerticalSplitterCollapseProps;
 
 /**
- * VerticalSplitterResizeParams interface. This defines the parameters of a resize operation. When
- * invert is true, the mouse delta is subtracted from the starting width instead of being added to
- * it, which inverts the resize operation.
+ * VerticalSplitterResizeParams interface. This defines the parameters of a resize operation.
  */
 export interface VerticalSplitterResizeParams {
 	readonly minimumWidth: number;
 	readonly maximumWidth: number;
 	readonly columnsWidth: number;
 }
 
-/**
- * ResizeParams interface. Represents a resize that's in progress.
- */
-interface ResizeParams {
```

Comment:
We no longer require this interface because the react state variable for the resize params is no longer needed for the new approach to handle resizing.